CHANGES.rst

```diff
@@ -4,7 +4,7 @@
 New Features
 ------------
 
-- Infrastructure to support auto-updating plugin results. [#2680]
+- Infrastructure to support auto-updating plugin results, not yet available to users. [#2680, #2834]
```

Comment:
we could instead just remove this entry altogether - any preferences?

Reply:
👍 to remove change log entry. If user cannot use, it does not need an entry. Thanks!